### DISCUSSION TITLE
Could Section List be foldable?

### DISCUSSION THREAD

**Discussion by radiorz** [GeekyAnts/NativeBase#5178] [Feature request]
### Description

The Section list is good. it will render some first and render the other for performance.
But if the section is foldable, the performance must be better.

### Problem Statement

The Section list is good. it will render some first and render the other for performance.
But if the section is foldable, the performance must be better.

### Proposed Solution or API

<SectionList>

**Comment by Viraj-10** [comment]
Hi @radiorz, Thanks for the suggestion. We will consider this feature request in a future sprint.

**Comment by Viraj-10** [comment]
Hi @radiorz, This seems like the `Accordion` component and it's on the road map.
